__test__/backend.test.js

```diff
@@ -173,564 +177,570 @@ async function createCampaign(user, title, description, organizationId) {
 
 // graphQL tests
 
-beforeAll(
-  async () => await setupTest(),
-  global.DATABASE_SETUP_TEARDOWN_TIMEOUT
-);
-afterAll(
-  async () => await cleanupTest(),
-  global.DATABASE_SETUP_TEARDOWN_TIMEOUT
-);
-
-it("should be undefined when user not logged in", async () => {
-  const query = `{
-    currentUser {
-      id
-    }
-  }`;
-  const context = getContext();
-  const result = await graphql(mySchema, query, rootValue, context);
-  const data = result;
+describe("graphql test suite", async () => {
```

Comment:
@schuyler1d this is a huge diff of this file.  The only new code is at the end of the file -- the test I added for getAvailableActionHandlers.  The rest of it is because I wrapped the test in a `describe` block to avoid test pollution.

**If you hide whitespace changes it's a lot less scary!**